pallets/liquidity-mining/src/tests/mock.rs

```diff
@@ -409,10 +414,6 @@ impl DustRemovalAccountWhitelist<AccountId> for Whitelist {
 	type Error = DispatchError;
 
 	fn add_account(account: &AccountId) -> Result<(), Self::Error> {
```

Comment:
It started to fail because new genesis build test was introduced, and we are trying to add the same account multiple times. The issue is in the mock file because we use one variable to track whitelisted accounts for 3 pallet instances. It cannot fail in the tests, but in runtime it can fail.